boltdesignsystem/bolt
# Hotfix: Yarn error and misplaced icons

## Jira

n/a

## Summary

Fixes yarn error and misplaced icons.

## Details

### Yarn error
`@bolt/testing-utils` was [not incremented](https://github.com/boltdesignsystem/bolt/blob/master/package.json#L114) from v4 to v5 in the v5.0.0 release and it triggers a yarn error when you try to install the site locally. The error is related to having multiple versions of some dependency and yarn not being able to properly resolve it:
```
error An unexpected error occurred: "expected workspace package to exist for \"glob-parent\"".
```

`glob-parent` I suspect is a dependency of a dependency of `@bolt/testing-utils`. Incrementing to v5 fixes it.

This error does not seem to happen on yarn 1.19.x, which is the version Travis runs on. It was only caught locally where most of us run 1.22.x.

> Perhaps we could lock Travis to 1.22.x to catch this in the future. Or, we ignore for now and upgrade to [Yarn 2](https://github.com/yarnpkg/yarn/issues/7807#issuecomment-817178104) in the next build which doesn't seem to experience this error ([I'm leaning towards the latter](https://spin.atomicobject.com/2021/02/17/switch-yarn-2/)).

### Misplaced icons
In https://github.com/boltdesignsystem/bolt/pull/2301 some icons were reverted. When we merged in newer icon updates we did not resolve these reverted icons which now appeared in the wrong directory. This PR moves them where they belong.

## How to test

- Run `yarn clean`, `yarn setup`, and `yarn start` to verify PL builds as expected
- Verify new icons have been added back and appear as expected in the Icon Element docs
